src/extension/ddappsec.c

```diff
@@ -412,9 +412,20 @@ static PHP_MINFO_FUNCTION(ddappsec)
     PUTS("(c) Datadog 2021\n");
     php_info_print_box_end();
 
+    char *state;
+    if (DDAPPSEC_G(enabled) == ENABLED) {
+        state = "Enabled";
+    } else if (DDAPPSEC_G(enabled) == DISABLED) {
+        state = "Disabled";
+    } else {
+        state = "Not configured";
+    }
+
     php_info_print_table_start();
-    php_info_print_table_row(2, "Datadog AppSec support",
-        DDAPPSEC_G(enabled) == ENABLED ? "enabled" : "disabled");
+    php_info_print_table_row(2,
+        "Datadog AppSec status managed by remote config",
+        DDAPPSEC_G(enabled_by_configuration) == NOT_CONFIGURED ? "Yes" : "No");
+    php_info_print_table_row(2, "Datadog AppSec state", state);
```

Comment:
nit: the `Datadog AppSec status managed by remote config` seems a bit long, we can probably remove `Datadog AppSec` from both and call the first one `State managed by remote config` and the second one `Current state`. What do you think?

Reply:
Done